apps/sms/test/marionette/conversation_test.js

```diff
@@ -491,4 +503,83 @@ marionette('Conversation Panel Tests', function() {
       });
     });
   });
+
+  suite('Add recipients from thread', function() {
+    var conversationView;
+    setup(function() {
+      var thread = ThreadGenerator.generate({
+        numberOfMessages: 1
+      });
+      messagesApp.launch();
+
+      storage.setMessagesStorage([thread], ThreadGenerator.uniqueMessageId);
+      // Set empty contacts store.
+      storage.setContactsStorage();
+      var inboxView = new InboxView(client);
+      conversationView = inboxView.goToConversation(thread.id);
+      messagesApp.Composer.messageInput.sendKeys('message2');
+    });
```

Comment:
Before that, I need to *reproduce* the issue... which is not the case. I've not yet managed to make the integration test run in vagrant. I've tried Gasolin's foxbox, and now I'm trying with azasypkin's gaia-vagrant.

Btw I have the feeling `clang` is needed to run 'npm install' on gaia. Do you confirm that?